src/py/migraphx_py.cpp

```diff
@@ -144,6 +144,17 @@ struct npy_format_descriptor<half>
     static constexpr auto name() { return _("half"); }
 };
 
+template <>
+struct npy_format_descriptor<migraphx::fp8::fp8e4m3fnuz>
+{
+    static std::string format()
+    {
+        // following: https://docs.python.org/3/library/struct.html#format-characters
+        return "z";
```

Comment:
I don't see "z" in the commented link?

Reply:
Yes that is not correct.  I am not sure what should be correct format. Between "B", "b", or "c". I'll have to check.

Reply:
I've opened an issue: This thing needs to be tested out to see if numpy buffers are created correctly. 
https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/issues/2447